tests/foreman/cli/test_contentview.py

```diff
@@ -4065,6 +4065,46 @@ def test_version_info_by_lce(self, module_org, module_target_sat):
         )
         assert content_view['version'] == '1.0'
 
+    @pytest.mark.tier2
+    def test_show_all_repo_ids(self, module_org, module_product, module_target_sat):
+        """Hammer content-view list shows all repo ids
+
+        :id: 56d716f1-85cf-47d7-a8e6-29788374d318
+
+        :steps:
+            1. Add a large number of repositories to a CV
+            2. Publish the CV
+            3. Run hammer content-view list
+
+        :expectedresults: All IDs for the repo are listed, and not truncated
+
+        :BZ: 2141421
+
+        :customerscenario: true
+        """
+        # Create 30 repositories
+        repolist = []
+        id_list = []
+        for _ in range(30):
+            repo = module_target_sat.api.Repository(
+                product=module_product,
+                checksum_type='sha256',
+                mirroring_policy='additive',
+                download_policy='immediate',
+            ).create()
+            repolist.append(repo)
+            id_list.append(str(repo.id))
+        # Sync and publish all repos
+        cv = module_target_sat.api.ContentView(
+            organization=module_org, repository=repolist
+        ).create()
+        for repo in repolist:
+            repo.sync()
+        cv.publish()
+        # Run content-view list --name cv.name
+        list_info = module_target_sat.cli.ContentView.list({'name': cv.name})
+        assert (list_info[0]['repository-ids']) == ', '.join(id_list)
```

Comment:
I'm slightly worried if that order may change in future, consider using sets here
```suggestion
        assert set(list_info[0]['repository-ids']) == set(id_list)
```